pkg/envoy/proxy_test.go

```diff
@@ -146,3 +146,129 @@ func testOutputPath() string {
 		fmt.Sprintf("test-output-%x.json", time.Now().UnixNano()+int64(os.Getpid())),
 	)
 }
+
+func TestProxy_OverridingLoggerAndExtraArgs(t *testing.T) {
```

Comment:
Do these tests run for you locally?  They fail for me with `failed to unmarshal output file: invalid character '\r' in string literal`

(Also, CI is not running the unit tests)

Reply:
@curtbushko - I tried running my test on local - test is passing but this log line is printing. 
If I run `TestProxy` I get the same logs. How should I go about fixing `TestProxy` and my tests.
Also how to make. it running in CI?

Reply:
Now it is running fine - 
![Uploading Screenshot 2023-06-07 at 4.44.37 PM.png…]()

Reply:
I figured out what the problem was for me. I had the `brew install base64` instead of the GNU `brew install gbase64`. This was causing `fake-envoy` to not run at all....